CONTRIBUTING.md

```
@@ -1,130 +1,130 @@
-# 🌟 Contributing to GitHub Tracker
+   # 🌟 Contributing to GitHub Tracker
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)